modules/core/04-channel/keeper/events.go

```diff
@@ -137,3 +137,116 @@ func EmitTimeoutPacketEvent(ctx sdk.Context, packet exported.PacketI, channel ty
 		),
 	})
 }
+
+// EmitChannelOpenInitEvent emits a channel open init event
```

Comment:
Maybe for readability/code hygiene it's better if all these functions that you added go at the top of the file, since there are related to opening/closing a channel and the ones above right now are related to packet sending, which is something that happens after a channel is opened.

Reply:
Thanks, will fix